apps/marketing/content/blog/switching-from-slack-to-discord

```diff
@@ -1,6 +1,6 @@
 ---
 title: Switching to Discord
-description: Join us on documen.so/discord to share ideas, support, and memes.
+description: The Documenso community is growing and we feel the need to have a more community- and developer-friendly environment. We're switching to Discord.
```

Comment:
Is the `-` intentional here `community- and`?

Reply:
oops! fixed